build/build.cake

```diff
@@ -257,30 +261,21 @@ Task("Package")
     .IsDependentOn("InheritDoc")
     .Does(() =>
 {
-    // Invoke the pack target to generate the code to be packed.
-    MSBuildSolution("Pack", ("GenerateLibraryLayout", "true"), ("PackageOutputPath", nupkgDir));
-
-    foreach (var nuspec in GetFiles("./*.nuspec"))
-    {
-        var nuGetPackSettings = new NuGetPackSettings
-        {
-            OutputDirectory = nupkgDir,
-            Version = Version
-        };
-
-        NuGetPack(nuspec, nuGetPackSettings);
-    }
+    Information("\r\nCopy files needed for Nuget package into a directory and create the package");
+    StartPowershellFile("./Package-Nuget.ps1");
 });
 
 Task("UpdateHeaders")
     .Description("Updates the headers in *.cs files")
+    .IsDependentOn("Package")
     .Does(() =>
 {
     VerifyHeaders(true);
 });
 
 Task("StyleXaml")
     .Description("Ensures XAML Formatting is clean")
+    .IsDependentOn("UpdateHeaders")
```

Comment:
Why does this depend on UpdateHeaders?

Reply:
I'm really confused about how this script is handling dependencies. Maybe we should discuss in person.

---
In reply to: [280584967](https://github.com/windows-toolkit/SceneLoader/pull/1#discussion_r280584967) [](ancestors = 280584967)